### ISSUE TITLE
Opening Spoon's code on macOS removes the code

### ISSUE THREAD

**Issue by wkoszek** [Hammerspoon/hammerspoon#3592]
When I do:

```
cd ~/.hammerspoon/Spoons
cd spoon_name_here
open .
```

Hammerspoon appears to remove the source code files of that Spoon.

**Comment by cmsj** [comment]
Yes, this is an unfortunate bug, it's trying to re-import the .spoon bundle. The `open .` likely won't ever do what you want it to, because you're in a bundle, but at least we should not be deleting it.
